src/CoreBundle/private/rendering/objectviews/TCMSTableToClass/recordList.view.php

```diff
@@ -120,7 +120,7 @@ public function &Next()
      *
      * @return false|<?php echo $sClassName."\n"; ?>
      */
-    public function &Previous()
+    public function Previous(): false|<?php echo $sClassName."\n"; ?>
```

Comment:
Warum kommen da bei den drei Returntypes die Newlines dazu?

Reply:
Because php will not add a newline when you close the php tag - pushing the opening method parenthesis into the method name line